build.gradle

```diff
@@ -69,7 +69,12 @@ dependencies {
     implementation "org.springdoc:springdoc-openapi-data-rest:1.2.34"
     implementation "org.springdoc:springdoc-openapi-webmvc-core:1.2.34"
 
+    // driver for postgres
     implementation "org.postgresql:postgresql:42.2.1"
+    //driver for h2
+    implementation "com.h2database:h2:1.4.200"
```

Comment:
I know this was mentioned in the chat, but I do still not understand what makes this change necessary, sorry :D Is this a worksaround for something?

Reply:
For those who want to use the docker container out-of-the-box without additional overhead h2 is the default database. For productive use the configuration can be switched to postgres without any additional effort.